Tools/MGCB/BuildContent.cs

```diff
@@ -332,6 +332,8 @@ public void Build(out int successCount, out int errorCount)
                     _manager.CleanContent(sourceFile, destFile);                
             }
 
+            // TODO: Should we be cleaning copy items?  I think maybe we should.
```

Comment:
@cra0zy @nkast @Jjagg - I noticed we haven't been cleaning copy items.  It seems to me like we should.

Should i go implement this in a new PR?

Reply:
Indeed. Furthermore when you deleted then by hand and rebuild it will not output the filenames of the copied items.

Reply:
I'll implement "Clean" for copy items in a new PR then.